index.html

```diff
@@ -16,6 +16,8 @@
     </div>
 
     <div id="wrapper" style="display:none;">
+        {% include navigation.html %}
```

Comment:
The navigation does not actually work :(

Reply:
Made the changes, and it is working now!